sources/editor/Stride.GameStudio/Program.cs

```diff
@@ -224,7 +224,7 @@ private static async void Startup(UFile initialSessionPath)
             {
                 var message = "Could not find a compatible version of MSBuild.\r\n\r\n" +
                               "Check that you have a valid installation with the required workloads, or go to [www.visualstudio.com/downloads](https://www.visualstudio.com/downloads) to install a new one.\r\n" +
-                              "Also make sure you have the latest [.NET 6 SDK](https://dotnet.microsoft.com/) \r\n\r\n" +
+                              "Also make sure you have the latest [.NET 8 SDK](https://dotnet.microsoft.com/) \r\n\r\n" +
```

Comment:
Maybe separating the version into a constant wouldn't be a bad idea? What do you think?
```cs
$"Also make sure you have the latest [.NET {PackageSessionPublicHelper.netMajorVersion} SDK](https://dotnet.microsoft.com/) \r\n\r\n"
```

```cs
class PackageSessionPublicHelper{
   public const byte netMajorVersion = 8; //or NET_MAJOR_VERSION
   ....
}

Reply:
makes sense to me, is there a reason for using a byte over an int?

Reply:
Not much, it's simply the smallest integer data type avaiable (8 bit instead of 32 bits)

Reply:
But probably short or int will be good enough too 😅

Reply:
lol fair enough, I guess we are very far away from .NET 256. Should be added now, thank you for the idea!

Reply:
Yeah, We will worry about that in about 248 years lol